server/api/auth-test.js

```diff
@@ -0,0 +1,10 @@
+const { responses } = require('../responses')
+
+module.exports = {
+  method: 'get',
+  path: '/auth/test',
+  requireAuth: true,
+  handler: async () => {
+    return [responses.validToken]
```

Comment:
If you don't include a `body` in the response, just `return responses.validToken` also works.